src/DependencyInjection/HeimrichHannotAdvancedMemberExtension.php

```diff
@@ -15,12 +15,23 @@
 
 class HeimrichHannotAdvancedMemberExtension extends Extension
 {
+    const ALIAS = 'huh_advanced_member';
+
     public function load(array $configs, ContainerBuilder $container)
     {
         $loader = new YamlFileLoader(
             $container,
             new FileLocator(__DIR__.'/../../config')
         );
         $loader->load('services.yml');
+
+        $configuration = new Configuration();
+        $config = $this->processConfiguration($configuration, $configs);
+        $container->setParameter(self::ALIAS, $config);
+    }
+
+    public function getAlias()
+    {
+        return self::ALIAS;
     }
```

Comment:
what is this for? the constant is already public and you are using it in Configuration without getter

Reply:
No, this is a method you must override use a custom alias to configure your bundle ;) I just added the constant as I need the name in 3 positions in the code, but this is the most important :)

Reply:
https://symfony.com/doc/4.4/bundles/extension.html#manually-registering-an-extension-class FYI ;)